packages/strapi-admin/admin/src/containers/App/reducer.js

```diff
@@ -1,5 +1,6 @@
 // Shared constants
 import { fromJS } from 'immutable';
+import packageJSON from '../../../../package.json';
```

Comment:
I think this will break because we do not copy the pkg.json file in the .cache folder :/